WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalConnectingFailedUpdatePostalCode.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
```

Comment:
Instead of adding a modal for each error, since the only thing changing is the title, could we do something with `CardPresentModalRetryableError` instead? (Same comment basically for the incomplete address error PR.)

Reply:
Thanks, I didn't see that one. 

We can't use `CardPresentModalRetryableError` for the incomplete address error, because of #5552 – the behaviour needed is more than just to retry, before you retry it opens up a web view or takes you to Safari to fix the issue.

Really it would be nice if we could do the same thing for this error, but we don't get the URL from the API in that case. Because we don't have the URL, there's a bit more signposting here using the `bottomTitle` to tell the user what to do. We could use the `CardPresentModalRetryableError` in this case, but it's set up as a simple generic retryable error, so we'd need to make it more configurable. I'll add a convenience init so that we can use the existing class that way.

Reply:
I think it's awesome, since it will be one fewer modal to maintain, but I leave it up to you to decide if you'd like to do that.

Reply:
We agreed on a call that due to future plans, it would be clearer not to merge these.